plugins/single_plugins/expo.cpp

```diff
@@ -71,8 +74,18 @@ class wayfire_expo : public wf::plugin_interface_t
     } state;
 
     int target_vx, target_vy;
+    int initial_vx, initial_vy;
     std::unique_ptr<wf::workspace_wall_t> wall;
 
+    wf::option_wrapper_t<int> delay{"input/kb_repeat_delay"};
```

Comment:
Feels like we're duplicating code already written here: https://github.com/WayfireWM/wayfire/blob/master/plugins/scale/scale-title-filter.cpp#L19

Maybe we can extract that struct in `plugins/common/...` and re-use it whenever we need key repeat?

Reply:
Yes, that makes sense; I've moved it there